package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Plugin for Amplify to add support for live streaming. Made for Unicorn Trivia Workshop",
   "main": "index.js",
   "scripts": {
-    "postinstall": "node scripts/post-install.js",
+    "postinstall": "husky install && node scripts/post-install.js",

```

Comment:
This will fail on non-dev environment due to husky not being installed. Moving this to a setup-dev script might work better.

Reply:
Yes you are right, do you think I should add inside `post-install.js` a condition looking for the environment ?
I think the only way to know if the package is on non-dev/dev environment is to pass for instance `NODE_ENV=dev` when running `npm install -g` as mentioned inside the [CONTRIBUTING.md](https://github.com/awslabs/amplify-video/blob/master/CONTRIBUTING.md#manually-installing)

So, it will looks like this: `NODE_ENV=dev npm install -g`

What do you think ?